Bottom: 42b59cb7f554c623f97a8895f5dbbfcce916de9f
Top:    f3b4ce6b6c40fbbd6f3650ee0cf1dc64764fbd61
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:57:40 +0100

Refresh of hid-gtk-rework-view-flip-code

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index c8bc46c..27ceeb9 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1407,7 +1407,7 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
                              !new_comp_vis, !new_comp_vis);
     }
 
-   return 0;
+  return 0;
 }
 
 /* ------------------------------------------------------------ */
